docs/book/src/tasks/installation.md

```diff
@@ -34,8 +35,9 @@ Cluster API requires an existing kubernetes cluster accessible via kubectl, choo
 
 <h1>Warning</h1>
 
-[kind] is not designed for production use, and is intended for development environments only.
+**Minimum supported version of kind**: v0.6.x
```

Comment:
```suggestion
**Minimum KIND supported version**: v0.6.x
```

Reply:
I went for a link there.. calls out that `kind` is a specific thing without making it all caps.